validation/standalone-constraint-constraint.ttl

```diff
@@ -179,17 +181,38 @@
 .
 
 :DimensionRelation a sh:NodeShape ;
-    sh:property [
+    sh:property
+        [
         sh:path meta:dimensionRelation;
-        sh:property [
-            sh:path meta:relatesTo;
-            sh:nodeKind sh:IRI ;
-            sh:minCount 1;
-            sh:message "meta:dimensionRelation requires at least one meta:relatesTo";
-        ] ;
+            sh:property
+                [
+                    sh:path meta:relatesTo ;
+                    sh:nodeKind sh:IRI ;
+                    sh:minCount 1 ;
+                    sh:message "meta:dimensionRelation requires at least one meta:relatesTo" ;
+                ],
+                [
+                    sh:path
+                        (
+                            meta:relatesTo
+                            [ sh:inversePath sh:path ]
+                            [ sh:inversePath sh:property ]
+                        ) ;
+                    sh:minCount 1 ;
+                    sh:class cube:Constraint ;
+                    sh:message "value of meta:relatesTo must be a cube dimension" ;
+                ] ;
```

Comment:
With SHACL-SPARQL we would be able to check that the two dimensions actually belong to the same cube. As above, we merely verify that they the related dimension is use an **a cube**, not necessarily the same cube. That would potentially cause false-positive results if a graph contained multiple cubes. But since we typically only validate cubes individually, we should be covered

Reply:
@Rdataflow @l00mi WDYT?

Reply:
LGTM